app/app/fixtures/oauth_application.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+  "model": "oauth2_provider.application",
+  "pk": 1,
+  "fields": {
+    "created": "2020-02-29 21:30:30Z",
+    "updated": "2020-02-29 21:30:30Z",
+    "client_id": "VkHFfFlxnVCMYtYCWRxphbvPQUJjmER4cWtwqe7Q",
+    "client_secret": "xhLXNqXnOlkUiGCDFJLxQJjPb3CltLZNFZtzOsLgFfvwbYY60kYWRRBmqyjzfrnNyT6WWdIbfQEPpU854K2UiofmXtybFD1QkfIR9VVQyK3ttJpET7GormIBjK9CQRap",
```

Comment:
are this keys fake?

Reply:
Yes are installed with fixtures so it can connect out of the box.